models.yaml

```diff
@@ -0,0 +1,254 @@
+---
```

Comment:
rails convention seems to be `.yml` rather than `.yaml`

Reply:
Done